docs/en/Configuring-Alluxio-with-secure-HDFS.md

```diff
@@ -62,6 +62,13 @@ Then edit `alluxio-site.properties` file to set the under storage address to the
 Copy secure HDFS conf xml files (`core-site.xml`, `hdfs-site.xml`, `mapred-site.xml`, `yarn-site.xml`) to
 `${ALLUXIO_HOME}/conf/`
 
+Optionally, you can set jvm-level system properties for customized Kerberos configurations:
```

Comment:
doest that mean, if you DO have customzied Kerberos configs, you are forced to reset these two properties then?